cmd/gonomics/gonomics.go

```diff
@@ -64,10 +69,41 @@ func getBin() (path string, binExists map[string]bool) {
 	return path, binExists
 }
 
-// tryPath tests a path for the presence of intervalOverlap (arbitrary chosen command)
+// getSrc returns the path to the source code files for commands
+func getSrc() string {
+	var expectedPath string
+	var cachedSrcPath string = getCachedSrcDir()
+	gopath := os.Getenv("GOPATH") + "/src/github.com/vertgenlab/gonomics/cmd/"
+	godefault := os.Getenv("HOME") + "/go/src/github.com/vertgenlab/gonomics/cmd/"
+
+	switch {
+	case cachedSrcPath != "" && tryPathDir(cachedSrcPath):
```

Comment:
That is a good point. I think it is a fairly minor change so I am going to go ahead and merge this PR and open a new one in a few minutes with just this addition.

Reply:
So this turns out to be non-trivial. This switch case is only a secondary decision on whether to use the cache or the source. The core branching point is at the start of getGonomicsCmds function where if the cache is full, we use the cache. To add the functionality you suggest we would need to define a trigger to decide whether we should use the cache or the source. Alternatively we would just always use the source when present, but running the cmd from source takes ~30s which was the initial impetus for making a cache, so not really a viable option. The only reasonable trigger I can think of is to check if any of the source code has been modified since the last time the gonomics cmd was built. That would require running os.Stat on each file in the source code before executing the command. That is a valid option, but it would probably add a couple seconds to the runtime and be quite a bit of work to code. I am in favor of just asking developers to rebuild the gonomics cache manually if they want to use uncommited commands which I don't think is too big an ask since I suspect most devs will rarely use the gonomics cmd anyways. If anyone feels strongly about this, I am happy to add this as a todo for me.